src/viam/robot/client.py

```diff
@@ -517,8 +519,8 @@ async def stop_all(self, extra: Dict[ResourceName, Dict[str, Any]] = {}):
         Cancel all current and outstanding operations for the robot and stop all actuators and movement
 
         Args:
-            extra (Dict[ResourceName, Dict[str, Any]]): Any extra parameters to pass to the components' ``stop`` methods, keyed on the
-                                                        component's ``ResourceName``
+            extra (Dict[viam.proto.common.ResourceName, Dict[str, Any]]): Any extra parameters to pass to the components' ``stop`` methods,
+                keyed on thecomponent's ``ResourceName``
```

Comment:
(nit) `thecomponent` -> `the component`.

Also, `s/component/resource`? I'd expect that `stop_all` would also stop services.